internal/build/image_builder.go

```diff
@@ -402,8 +410,8 @@ func messageIsFromBuildkit(msg jsonmessage.JSONMessage) bool {
 	return msg.ID == "moby.buildkit.trace"
 }
 
-func getDigestFromBuildOutput(ctx context.Context, reader io.Reader) (digest.Digest, error) {
-	aux, err := readDockerOutput(ctx, reader)
+func (l *dockerImageBuilder) getDigestFromBuildOutput(ctx context.Context, reader io.Reader) (digest.Digest, error) {
```

Comment:
we should probs eventually change the name of this receiver, given the struct name change.

Reply:
isn't it still a docker image builder? It's an image builder that builds docker images?

Reply:
ah i mean the letter `l` no longer stands for anything :P

Reply:
ohhh got it, yes will change